src/sagemaker/jumpstart/types.py

```diff
@@ -1730,3 +1746,37 @@ def __init__(
         self.nearest_model_name = nearest_model_name
         self.data_input_configuration = data_input_configuration
         self.skip_model_validation = skip_model_validation
+
+
+class HubArnExtractedInfo(JumpStartDataHolderType):
```

Comment:
Not really necessary for this PR (probably for my PR), we should think of locations for hub-related types/enums/constants/... i.e. if we should keep it under jumpstart/ or jumpstart/curated_hub

Reply:
+1 agreed, I think if the enum/type/constant/etc are used in the curated_hub.py file, we should keep those there. If they're used in others as well, we can keep it in the top level. WDYT?